From 37bb6af863dbedcef96f7a98e1e26ba9f50d34c8 Mon Sep 17 00:00:00 2001
From: Jami Kettunen <jami.kettunen@protonmail.com>
Date: Sat, 2 Apr 2022 18:41:40 +0300
Subject: [PATCH 117/127] arm64: dts: qcom: msm8998-oneplus-common: Add fuel
 gauge

The OnePlus 5/5T feature a BQ27411 fuel gauge for reading the battery
stats.
---
 arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi
index 5fbe0b77d5c7..7964114f656e 100644
--- a/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/msm8998-oneplus-common.dtsi
@@ -270,6 +270,15 @@ cts {
 	};
 };
 
+&blsp2_i2c1 {
+	status = "okay";
+
+	fuel-gauge@55 {
+		compatible = "ti,bq27411";
+		reg = <0x55>;
+	};
+};
+
 &blsp2_i2c3 {
 	status = "disabled";
 
-- 
2.38.1

